k8s/tilt-nats-values.yaml

```diff
@@ -0,0 +1,19 @@
+nats:
```

Comment:
qq: can we use the config file? https://github.com/guacsec/guac/blob/main/container_files/nats/js.conf? 

If its hard to use it, can we maybe sync up values to 64M?

Reply:
no easy way to re-use that file via the Helm chart for NATS, so I opted to update it instead

Reply:
@j-white just need to sign the commit and should be good to merge!